src/main.c

```diff
@@ -17,6 +17,7 @@
 #include "modules.h"
 #include "parse.h"
 #include "util.h"
+#include "libgen.h"
```

Comment:
This should be in the first list of includes, as `#include <libgen.h>`, since it is a system header.